composer.json

```diff
@@ -46,15 +46,15 @@
         "drupal/config_update": "^1.6",
         "drupal/context": "^4.0@beta",
         "drupal/context_groups": "^2.x-dev",
-        "drupal/core": "^9",
-        "drupal/core-composer-scaffold": "^9",
-        "drupal/core-project-message": "^9",
-        "drupal/core-recommended": "^9",
+        "drupal/core": "^9.3.14",
+        "drupal/core-composer-scaffold": "^9.3.14",
+        "drupal/core-project-message": "^9.3.14",
+        "drupal/core-recommended": "^9.3.14",
         "drupal/ctools": "^3.4",
         "drupal/default_content": "^2.0@alpha",
         "drupal/default_content_deploy": "^1.0@beta",
         "drupal/ds": "^3.5",
-        "drupal/email_registration": "^1.1",
+        "drupal/email_registration": "^1.x-dev",
```

Comment:
Not sure we should use dev version. Maybe it's OK, but need to see the diff.

Reply:
I pushed it with other branch and it's already in master:
354927acfe0a045ad629405399f6b246f3dc3e06
86 - Use stable version of Email Registration

I merged master in this pull request